_posts/data-announces-formation.md

```diff
@@ -51,7 +51,7 @@ Tyler Wright, Interim Co-Chair
 Connor Hulla, Interim Co-Chair  
 Dallas Area Transit Alliance (DATA)  
 Email: [info@ridewithdata.org](mailto:info@ridewithdata.org)   
-Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org)  
+Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org)  and [ridewithdata.org](https://ridewithdata.org)  
```

Comment:
```suggestion
Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org) or [ridewithdata.org](https://ridewithdata.org)  
```

Reply:
Fixed in commit https://github.com/CurtisGreen/dallas-area-transit-alliance/pull/13/commits/acce2d4e67fd760fae86959d5709ceb6f9517da3